projects/plugins/crm/includes/jpcrm-templating-placeholders.php

```diff
@@ -1755,6 +1755,27 @@ public function replace_placeholders(
 					
 					}
 
+					// If this is a Quote date key and is not set (Quote accepted or last viewed), let's print out a message saying the quote isn't accepted or viewed.
+					$possible_empty_quote_keys = array( 'quote-accepted', 'quote-accepted_date_str', 'quote-accepted_datetime_str', 'quote-lastviewed', 'quote-lastviewed_datetime_str', 'quote-lastviewed_date_str' );
+					if ( in_array( $key, $possible_empty_quote_keys, true ) ) {
+						// 82799 is 23hrs, 59mins, 59secs on Jan 1, 1970 (so a blank timestamp, allowing for systems to update the time within that day for any reason)
+						if ( empty( $replace_with ) || strtotime( $replace_with ) <= 82799 ) {
+							$replace_with = str_starts_with( $key, 'quote-accepted' ) ? __( 'Quote not yet accepted', 'zero-bs-crm' ) : __( 'Quote not yet viewed', 'zero-bs-crm' );
+						}
+					}
+
+					// We want to check if a quote field is a unix timestamp, so this light approach will check if the value is numeric initially to lower the searches,
```

Comment:
This is not a Unix timestamp...perhaps that's what lead to the confusion and the desire to add (undocumented) `_DATETIME_STR` and `_DATE_STR` placeholders?

Reply:
There's inconsistency with how we show things here vs. with Unix timestamps elsewhere. In other objects, we show the following placeholders:

* `##SOMETHING_DATE##` <-- UTS
* `##SOMETHING_DATE_DATE_STR##` <-- formatted date string
* `##SOMETHING_DATE_DATETIME_STR##` <-- formatted datetime string

Reply:
That's now fixed in a follow-up commit.